tests/unit/compile_test.py

```diff
@@ -35,6 +35,34 @@ def test_abi_algobank():
         assert contract.dictify() == expected_contract
 
 
+def test_abi_vote():
```

Comment:
```suggestion
def test_abi_poll():
```

nit - slightly more in line with the current naming